shared/js/media/media_utils.js

```diff
@@ -98,5 +106,12 @@ var MediaUtils = {
       return MediaUtils.binarySearch(array, element, comparator, from, mid);
     else
       return MediaUtils.binarySearch(array, element, comparator, mid + 1, to);
+  },
+
+  shortDescription: function(timestamp, video) {
+    var date = new Date(timestamp);
+    var dtf = this._getDateTimeFormat();
+    return dtf.localeFormat(
+      date, this._(video ? 'videoDescriptionShort' : 'imageDescriptionShort'));
```

Comment:
Same nit here about line wrapping.  And if it was me, I'd declare  var desc and do the this._ call and the conditional operator in the variable initalizer, to make the localeFormat call nice and clean.